From: Alexis Murzeau <amubtdx@outlook.fr>
Date: Tue, 11 Jul 2017 00:49:30 +0200
Subject: Makefile.in: fix recursive part

---
 Makefile.in | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/Makefile.in b/Makefile.in
index c4f8ce7..76ed0da 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -25,14 +25,10 @@ endif
 # (who knows?), MAKEFLAGS contains 'Rr' instead of '-Rr', while adding
 # '-Rr' to MAKEFLAGS adds it literaly ( and does not add 'Rr' )
 # Further: quad test because the flags 'rR' and '-rR' can be reordered.
-ifeq ($(filter Rr,$(MAKEFLAGS)),)
-ifeq ($(filter -Rr,$(MAKEFLAGS)),)
-ifeq ($(filter rR,$(MAKEFLAGS)),)
-ifeq ($(filter -rR,$(MAKEFLAGS)),)
+ifeq ($(findstring Rr,$(MAKEFLAGS)),)
+ifeq ($(findstring rR,$(MAKEFLAGS)),)
 CT_MAKEFLAGS += -Rr
-endif # No -rR
 endif # No rR
-endif # No -Rr
 endif # No Rr
 
 # Remove any suffix rules
